hbase-oss/pom.xml

```diff
@@ -346,11 +370,35 @@
       <artifactId>slf4j-log4j12</artifactId>
       <version>${slf4j.version}</version>
       <scope>test</scope>
+      <exclusions>
```

Comment:
Isn't `slf4j-log4j12` a log4j-1 related dependency? Can it be removed completely and excluded where it comes transitively from hadoop/zookeeper?